crates/core/types/genesis.rs

```diff
@@ -120,19 +122,19 @@ impl Genesis {
             extra_data: Bytes::new(),
             prev_randao: self.mixhash,
             nonce: self.nonce,
-            base_fee_per_gas: 0,
-            withdrawals_root: None,
-            blob_gas_used: None,
-            excess_blob_gas: None,
-            parent_beacon_block_root: None,
+            base_fee_per_gas: INITIAL_BASE_FEE,
+            withdrawals_root: Some(compute_withdrawals_root(&[])),
+            blob_gas_used: Some(0),
+            excess_blob_gas: Some(0),
+            parent_beacon_block_root: Some(H256::zero()),
         }
     }
 
     fn get_block_body(&self) -> BlockBody {
         BlockBody {
             transactions: vec![],
             ommers: vec![],
-            withdrawals: None,
+            withdrawals: Some(vec![]),
```

Comment:
Im surprised these changes don't break any of the spec test. Does the rlp and the state root remain equal with these values (empty vec vs None, 0 vs None)?

Reply:
The state root shouldn't be affected (as it is built from the accounts in the alloc). The rlp does change, and do we need it to change as with the current version block validation failed when running with kurtosis as the genesis block's hash didn't match the parent_block_hash of the following block